kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxy.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kroxylicious;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+import org.junit.jupiter.api.TestInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.Pod;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.Environment;
+import io.kroxylicious.systemtests.resources.manager.ResourceManager;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyConfigTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyDeploymentTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyServiceTemplates;
+import io.kroxylicious.systemtests.utils.DeploymentUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Kroxy.
+ */
+public class Kroxy {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Kroxy.class);
+    private final String deploymentNamespace;
+    private String containerImage = "quay.io/kroxylicious/kroxylicious-developer:" + Environment.KROXY_VERSION;
```

Comment:
We said elsewhere that we are not going to hardcode the use of quay.io.  Quay.io as a default is okay, but the developer should be free to use alternative container registries (e.g. docker.io)

Reply:
done